packages/sdk/src/sdk/index.ts

```diff
@@ -26,6 +26,7 @@ export * from './api/albums/types'
 export * from './api/playlists/types'
 export * from './api/tracks/types'
 export * from './api/users/types'
+export * from './middleware'
```

Comment:
Oops this isn't actually used. But I'm going to leave it in because it'll be needed for making the IdentityApi class anyway.